app/views/tag/show/_nav_tabs.html.erb

```diff
@@ -4,12 +4,9 @@
     <% if params[:action] == "show" %>
       <% unless params[:id].match("question:") %>
         <li class="nav-item">
-          <a class="nav-link <% if @node_type == "note" %> active<% end %>" href="/tag/<%= params[:id] %>">
+          <a class="nav-link <% if @node_type == "note" %>active<% end %>" href="/tag/<%= params[:id] %>/author/<%= params[:author] %>">
             <i class="fa fa-file"></i> 
-            <span class="d-lg-inline"> 
-              <span class="d-none d-md-inline"><%= t('tag.show.research_notes') %></span>
-              <span class="d-none d-sm-inline badge badge-primary"><%= params[:counts][:posts] %></span>
-            </span>
+            <span class="d-lg-inline"><%= raw t('tag.show.research_notes') %></span>
           </a>
         </li>
       <% end %>
```

Comment:
@2201monesh The changes you made above are not the ones you were supposed to make. I'd suggest taking a look at #9644 again.

Reply:
@TildaDares thank you and I am looking at it once again